Dockerfile

```diff
@@ -1,20 +1,31 @@
-# Use an the sapmachine runtime as a parent image
-FROM sapmachine/jdk11
+FROM gradle:7.4.2-jdk11 AS builder
 
-# The build NeonBee version
-ARG NEONBEE_VERSION
+RUN mkdir app
+WORKDIR /app
+COPY . /app
+RUN gradle -x test -x spotlessJava \
+    -x spotlessCheck -x spotlessJavaCheck \
+    -x spotbugsMain -x spotbugsTest \
+    -x pmdMain -x pmdTest \
+    -x checkstyleMain -x checkstyleTest \
+    -x violations \
+    -x reportCoverage \
+    -x testJar -x testSourcesJar \
+    -x javadoc -x javadocJar -x testJavadoc -x testJavadocJar \
+    --no-daemon clean build
```

Comment:
The gradle build steps hangs on my Mac M1.

 nerdctl build -t neonbee-core . --platform=linux/amd64

Reply:
Seems to be for every environment individual. It seems to work for the most people and also (most importantly) for the GitHub Workflow pipeline building the release package.